scripts/data_files/vs2013-main-template.vcxproj

```diff
@@ -91,6 +91,9 @@ INCLUDE_DIRECTORIES
     <Link>
       <SubSystem>Windows</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
+      <ShowProgress>NotSet</ShowProgress>
+      <AdditionalDependencies>bcrypt.lib;kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comdlg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
I'm not an expert in hand-coding vcxproj'es, and I know this normally ends up in tears, but I have to ask why have you added the full block of dependancies here, when there were no dependancies before.

Reply:
Removed and simplified to an one line

    <AdditionalDependencies>bcrypt.lib;%(AdditionalDependencies)</AdditionalDependencies>